### PR TITLE
[#967] Refactored Inbox views to use user UUID, cleaned code

### PR DESCRIPTION
**Pull request by Bartvaderkin**
With "cleaned code" I mean I renamed all the `self.me` / `me`'s I ran into.

### CODE REVIEW

**Comment by Bartvaderkin** [comment]
I added a commit that disables the `async_selector` on the form-submit, because it replaces then whole `.messages` element including the form and emoji/file widget without re-hydrating them. 

I tried using the same technique we used for the Dropdown component (in `src/open_inwoner/js/components/index.js`) but the Emoji and MessageFile javascript code isn't as clean: it keeps global references and event listeners so the htmx swap can't remove everything.

File: src/open_inwoner/accounts/views/inbox.py [hunk 1]
```diff
@@ -126,24 +142,31 @@ def get_form_kwargs(self):
...
         return HttpResponseRedirect(f"{url}#messages-last")
 
     def get(self, request, *args, **kwargs):
-        """Mark all messages as seen for the receiver"""
         context = self.get_context_data()
 
-        # Redirect to the end of page.
-        # Redirecting to a hash doesn't work, so we need to change the url.
-        # Alter URL with redirected query param in order to go to the last message (#messages-last).
+        """
```
Comment (vaszig) [inline src/open_inwoner/accounts/views/inbox.py:156]: By removing this part here it seems that the `message.seen`  is never updated to True. That's why Mijn berichten (navigation bar) is not updated with the number of seen=False messages.
image

**Review by vaszig — CHANGES_REQUESTED** [review]

File: src/open_inwoner/accounts/views/inbox.py (hunk 1 above)
Comment (Bartvaderkin) [reply src/open_inwoner/accounts/views/inbox.py:156]: Good catch. I'll need to fix it and see about the tests.

Comment (vaszig) [reply src/open_inwoner/accounts/views/inbox.py:156]: Tested again the code and works as expected. Maybe something was wrong with the browsers.

**Review by vaszig — APPROVED** [review]

**Review by alextreme — APPROVED** [review]
Approving in order to get v1.1 ready, thanks guys

**Pull request merged**